### DISCUSSION TITLE
[question] actually a few

### DISCUSSION THREAD

**Discussion by noctuum** [theAsmodai/metamod-r#82] [Q&A] [answered]
hello i've got a few questions
please pardon me if they might be silly

1. does this metamod better than [p39](https://github.com/SamVanheer/Metamod-P-CMake)?
any criteria (faster/more stable/whatever)

2. does it support Adrenaline Gamer?
if no, do you plan to add support?

3. does it support Sven Coop 5.xx ?
if no, do you plan to add support?

thanks in advance

**Comment by hajimura** [comment]
1.  Metamod-r is faster
2. Metamod-r supports AG
3. I think, yes, but you must manually load you need game binary (.dll/.so). Read this: https://github.com/theAsmodai/metamod-r/blob/master/metamod/extra/config.ini#L31

**Comment by stamepicmorg** [comment]
**krangm**
> does it support Adrenaline Gamer?
> if no, do you plan to add support?
> 
> does it support Sven Coop 5.xx ?
> if no, do you plan to add support?

**theAsmodai** 
> I think, yes, but you must manually load you need game binary (.dll/.so). Read this: 

@krangm, now Metamod-r is  [supported](https://github.com/theAsmodai/metamod-r/blob/master/metamod/src/game_support.cpp) AG and SvenCoop. See [README.MD](https://github.com/theAsmodai/metamod-r/blob/master/README.md) file with compablity list.

**Comment by noctuum** [comment]
@stamepicmorg OpenAG is just a client, doubt its related to metamod anyhow. In common yes, AG servers work fine with metamod-r but cant tell same about Sven Co-op 5.xx. At least on default HLSP maps server becomes critically unstable with metamod (any, not only -r). Server fps drops to ~ 20, player pings grow to nearly +150 from what they had without metamod. Feels awful and unplayable.
Guess this is more Sven Co-op problem rather than metamod but still cant say game is 'fully supported'.

**Comment by stamepicmorg** [comment]
@krangm

Maby if...

>HLSP maps server becomes critically unstable with metamod (any, not only -r)

....problem not at metamods? 

Anyway.. you have no add:
* info about unstable work
* crashdumps
* any logs

Can u please write:
* your os
* version of svencoop
* version of metamod(s)
* problem maps
* server's build\rehlds ver.
* filename of svencoop


and  i will try to reproduce it.
 
***

How can mainteiners `add support` without any help from otherside?  I think more rational is ask about bugs in `Sven Co-op 5.xx.` developers of `Sven Co-op 5.xx.`.

**Comment by stamepicmorg** [comment, marked as answer]
@krangm hello! problem resolved [here](https://github.com/theAsmodai/metamod-r/issues/45).
